src/main/java/uk/ac/ebi/eva/pipeline/io/mappers/VariantVcfFactory.java

```diff
@@ -333,13 +342,21 @@ protected void parseInfo(Variant variant, String fileId, String studyId, String
                         break;
                     case "AF":
                         // TODO For now, only one alternate is supported
-                        String[] frequencies = splits[1].split(",");
-                        file.addAttribute(splits[0], frequencies[numAllele]);
+                        if(splits[1].equals("0")) {
```

Comment:
Please follow the same indentation style as the rest of the code (4 spaces instead of one tab).